tests/test_conformer.py

```diff
@@ -65,7 +65,11 @@ def get_output_shape(input_shape, cfg, **kwargs):
     # with key padding mask
     input_shape = [4, 15, 32]  # B,T,F
     cfg = ConformerMHSAV1Config(32, 8, 0.2, 0.3)
-    assert get_output_shape(input_shape, cfg, key_padding_mask=torch.randint(0, 2, input_shape[:2]) > 0) == [4, 15, 32]
+    assert get_output_shape(input_shape, cfg, sequence_mask=(torch.randint(0, 2, input_shape[:2]) > 0)) == [
+        4,
+        15,
+        32,
+    ]
```

Comment:
```suggestion
    assert get_output_shape(input_shape, cfg, sequence_mask=(torch.randint(0, 2, input_shape[:2]) > 0)) == [4, 15, 32]
```
this line is even shorter than the original one.. black should not be overzealous here